test/testutils/manager.go

```diff
@@ -40,6 +42,7 @@ func StartManager(ctx context.Context, t *testing.T, mgr manager.Manager) {
 func SetupManager(t *testing.T, cfg *rest.Config) (manager.Manager, *watch.Manager) {
 	t.Helper()
 
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Is this for debugging or meant to be in product code?

Reply:
otherwise we don't have logs during a test failure.

Reply:
going to resolve this for now; feel free to open if still a concern.